json-schema/json-schema/src/main/scala/endpoints4s/NumericConstraints.scala

```diff
@@ -0,0 +1,87 @@
+package endpoints4s
+
+/** Possible restrictions on the numeric value used. Needs an instance for [[Ordering]] to check whether values are
+  * valid w.r.t. to the properties.
+  * @group operations
+  */
+final class NumericConstraints[A] private (
+    val minimum: Option[A] = None,
+    val exclusiveMinimum: Option[Boolean] = None,
+    val maximum: Option[A] = None,
+    val exclusiveMaximum: Option[Boolean] = None,
+    val multipleOf: Option[A] = None
+)(implicit val ord: Ordering[A], mult: MultipleOf[A])
```

Comment:
Why do we need `ord` to be a `val`?

Reply:
We don't, not sure why I did that. I'll remove it. I'll also use `implicitly` for `Numeric`, maybe that will either fix the build problem, or at least give a better error message.

Reply:
Great, yes I think removing the `val` here, and using `implicitly` in the test failure should help.